src/lavinmq/vhost.cr

```diff
@@ -58,6 +58,18 @@ module LavinMQ
       @shovels = ShovelStore.new(self)
       @upstreams = Federation::UpstreamStore.new(self)
       load!
+      spawn check_consumer_timeouts_loop
+    end
+
+    private def check_consumer_timeouts_loop
+      loop do
+        sleep Config.instance.consumer_timeout_loop_interval
+        @connections.each do |c|
+          c.channels.each_value do |ch|
+            ch.check_consumer_timeout
+          end
+        end
+      end
```

Comment:
We should make this fiber close down gracefully when vhost is closed. It's most likely necessary for MT.